packages/terra-data-grid/src/proptypes/DataGridPropTypes.js

```diff
@@ -2,8 +2,8 @@ import rowShape from './rowShape';
 import cellShape from './cellShape';
 import { columnShape, SortIndicators } from './columnShape';
 
-const WorklistDataGridPropTypes = {
+const DataGridPropTypes = {
   cellShape, columnShape, rowShape, SortIndicators,
 };
 
-export default WorklistDataGridPropTypes;
+export default DataGridPropTypes;
```

Comment:
Do we really need this object?  It is just grouping import statements for us, correct?  These prop types will not live with the data grid long term so it would be nice to be able to remove this additional object to maintain.  Once we create the table, this would be another object that we would need to rename.

Reply:
Removed